examples/v2_banpeers.rs

```diff
@@ -0,0 +1,60 @@
+//! Example of banning functionality of the node.
+use std::str::FromStr;
+
+use anyhow::{ensure, Context};
+use clap::AppSettings;
+use concordium_rust_sdk::{types::network::PeerToBan, v2};
+use structopt::StructOpt;
+
+#[derive(StructOpt)]
+struct App {
+    #[structopt(
+        long = "node",
+        help = "GRPC interface of the node.",
+        default_value = "http://localhost:10001"
+    )]
+    endpoint: tonic::transport::Endpoint,
```

Comment:
Please use the re-exported Endpoint `concordium_rust_sdk::endpoints::Endpoint`.

That way people can just copy-paste the example and just have concordium-rust-sdk as a dependency.